msvc/eglibarm64.vcxproj

```diff
@@ -0,0 +1,213 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Where is this project needed? Beyond `msvc/eglib.vcxproj`?

Reply:
But why is eglib.vcxproj not enough?

Reply:
In case what Jem said isn't entirely clear, here's a diagram:

                       -----------> mono-bdwgc (ARM64) -> eglib (ARM64)
                     /
    Mono.sln
                     \
                       -----------> genmdesc (x64) -> eglib (x64)

 The way I understand it it's because both x64 and ARM64 versions of the library are needed in the same build graph and MSBuild doesn't support that from a single project file.

Reply:
Thanks, makes sense!